### PR TITLE
Feat/svg icons

### PR DESCRIPTION
**Pull request by rivisic**
- Added new svg icon system.
- Current Icon component changed.
- Usage example: ` <Icon iconName="svg-icon-*" width="16px" height="16px"/>` where * is icon name

### CODE REVIEW

File: src/components/Icon/Icon.scss [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-	display: inline-block;
+    -webkit-font-smoothing: antialiased;
+    display: inline-block;
     font-family: icomoon;
     font-style: normal;
-	margin-right: 5px;
+    margin-right: 5px;
+}
+
+.svg {
```
Comment (vinkobedek) [inline src/components/Icon/Icon.scss]: maybe nest the svg class inside the icon class to limit the scope to the icon component.

File: src/components/Icon/Icon.tsx [hunk 2]
```diff
@@ -12,6 +12,14 @@ export const Icon: (props: IIconProps) => JSX.Element = (props: IIconProps) => {
             [props.iconName]: !customIcon
         }, [props.className]);
 
-    return <i { ...getNativeAttributes(props, htmlElementAttributes) } className={iconClassName} />;
+        if (props.iconName.startsWith('svg')) {
+            return <svg className = {'svg ' + props.iconName} width = {props.width} height= {props.height}>
```
Comment (vinkobedek) [inline src/components/Icon/Icon.tsx]: props.className is ignored for svg. Is this the desired behaviour?

File: src/components/Icon/Icon.scss (hunk 1 above)
Comment (rivisic) [reply src/components/Icon/Icon.scss]: Ok, but currently in the .icon class is many font-related properties which are not necessary for svgs.

File: src/components/Icon/Icon.tsx (hunk 2 above)
Comment (rivisic) [reply src/components/Icon/Icon.tsx]: It will be changed

File: src/components/Icon/Icon.scss (hunk 1 above)
Comment (vinkobedek) [reply src/components/Icon/Icon.scss]: indeed, you are correct, i mistakenly thought that there was some parent element, but it always returned <i/> directoly. how about removing the svg class and the using the following selectors
i.icon, svg.icon.

in the icon component we would the return either return 
```<i { ...getNativeAttributes(props, htmlElementAttributes) } className={iconClassName} />;```
or
```<svg className={iconClassName} />;```


with this the iconClassName will work as before and the component will respect the props.className.

**Pull request merged**